### PR TITLE
GoofMos: Heavily simplified atmospherics, Reagent Gases, Gas Mining, Pipe Removal

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
experimental: Gas has gotten a complete overhaul.
fix: Gas no longer requires a degree to understand.
feature: Pipes have been removed. It's faster and easier to have a shared gas-tank pulled from by vents and fed into by scrubbers.
/:cl:

### CODE REVIEW

**Comment by kevinz000** [comment]
NO LEAVE MY GASTISM ALONE

**Comment by Shadowflame909** [comment]
The madman did it. Right after I saw him being talked about in chat.

File: code/modules/atmospherics/gasmixtures/gas_types.dm [hunk 1]
```diff
@@ -59,7 +59,7 @@ GLOBAL_LIST_INIT(hardcoded_gases, list(/datum/gas/oxygen, /datum/gas/nitrogen, /
 /datum/gas/plasma
 	id = "plasma"
 	specific_heat = 200
-	name = "Plasma"
+	name = "Phoron"
```
[the remark this answers is not in the document]
Comment (ike709) [reply code/modules/atmospherics/gasmixtures/gas_types.dm]: 👎

File: code/modules/atmospherics/gasmixtures/gas_types.dm [hunk 2]
```diff
@@ -74,7 +74,7 @@ GLOBAL_LIST_INIT(hardcoded_gases, list(/datum/gas/oxygen, /datum/gas/nitrogen, /
 /datum/gas/hypernoblium
 	id = "nob"
 	specific_heat = 2000
-	name = "Hyper-noblium"
+	name = "Freon"
```
[the remark this answers is not in the document]
Comment (ike709) [reply code/modules/atmospherics/gasmixtures/gas_types.dm]: 👍

[the remark this answers is not in the document]
Comment (ACCount12) [reply code/modules/atmospherics/gasmixtures/gas_types.dm]: 👎

File: code/modules/atmospherics/gasmixtures/gas_types.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (ACCount12) [reply code/modules/atmospherics/gasmixtures/gas_types.dm]: 👎

File: code/modules/atmospherics/gasmixtures/gas_types.dm (hunk 2 above)
[the remark this answers is not in the document]
Comment (Iamgoofball) [reply code/modules/atmospherics/gasmixtures/gas_types.dm]: :+1:

File: code/modules/atmospherics/gasmixtures/gas_types.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (Iamgoofball) [reply code/modules/atmospherics/gasmixtures/gas_types.dm]: :+1:

**Comment by duncathan** [comment]
I'm not gonna lie, you caught me off guard with this one, not bad goof

**Pull request closed without merging**